node/node.d.ts

```diff
@@ -170,6 +170,7 @@ declare module NodeJS {
         removeAllListeners(event?: string): EventEmitter;
         setMaxListeners(n: number): void;
         listeners(event: string): Function[];
+        _events?:string[];
```

Comment:
But how can I get all events that are registed  to with the existing node.d.ts ? Sometimes you need all events no just the listeners(eventType)